cscs-checks/prgenv/helloworld.py

```diff
@@ -1,15 +1,13 @@
-import os
 from datetime import datetime
 
+import reframe as rfm
 import reframe.utility.sanity as sn
-from reframe.core.pipeline import RegressionTest
 
 
-class HelloWorldBaseTest(RegressionTest):
+class HelloWorldBaseTest(rfm.RegressionTest):
     def __init__(self, name, lang, linkage, **kwargs):
-        super().__init__('hello_world_%s_%s_%s' % (name, linkage, lang),
-                         os.path.dirname(__file__), **kwargs)
-
+        super().__init__()
+        self.name = 'hello_world_%s_%s_%s' % (name, linkage, lang)
```

Comment:
I wouldn't set the `name`. I think the automatic name generation can do pretty well here.